backend/app/tests/services/test_advisor_service.py

```diff
@@ -88,61 +80,61 @@ def get_previous_session() -> Session:
     return session
 
 
-class TestAdvisorService(unittest.TestCase):
-    def setUp(self) -> None:
-        self.advisor_service = AdvisorService()
-        self.llm_mock_response = get_mock_advisor_response()
-        self.previous_session = get_previous_session()
-        self.mock_session_feedback = get_mock_session_feedback()
+# class TestAdvisorService(unittest.TestCase):
```

Comment:
let's not forget to add them afterwards, but good that you commented them out for speed (as discussed)